packages/cli/api-importers/openapi/openapi-ir-to-fern/src/buildQueryParameter.ts

```diff
@@ -67,6 +67,13 @@ export function buildQueryParameter({
         queryParameterSchema.availability = convertAvailability(queryParameter.availability);
     }
 
+    if (typeof typeReference.value === "object" && "type" in typeReference.value) {
+        const detailedSchema = typeReference.value as RawSchemas.TypeReferenceDetailedSchema;
+        if (detailedSchema.validation !== undefined) {
+            queryParameterSchema.validation = detailedSchema.validation;
+        }
+    }
```

Comment:
nit: We can add a type guard function for this. That way we get the type narrowing on line 71 for free. Something along the lines of the following:

```typescript
function isRawTypeReferenceDetailedSchema(rawTypeReference: RawSchemas.TypeReference): rawTypeReference is RawSchemas.TypeReferenceDetailedSchema {
      return (rawTypeReference as RawSchemas.TypeReferenceDetailedSchema).type != null;
}
```

This is similar to the other `isRaw` helpers you'll see elsewhere (like [this](https://github.com/fern-api/fern/blob/ed573cc105a27a648e3ad5dfac2cc18a4de7e3ce/packages/cli/fern-definition/schema/src/utils/visitRawTypeDeclaration.ts#L42)). You can keep the function un-exported in this file for now if that's simpler.

Reply:
Implemented the suggested change.